libpod/container_internal.go

```diff
@@ -1132,6 +1133,16 @@ func (c *Container) pause() error {
 		return errors.Wrapf(define.ErrNoCgroups, "cannot pause without using CGroups")
 	}
 
+	if rootless.IsRootless() {
+		cgroupv2, err := cgroups.IsCgroup2UnifiedMode()
+		if err != nil {
+			return errors.Wrap(err, "failed to determine cgroupversion")
+		}
+		if !cgroupv2 {
+			return errors.Wrap(define.ErrNoCgroups, "can not pause containers on rootless containers with cgroup V1")
```

Comment:
We should discard pause here and warn instead of erroring. We can't break existing rootless installs.

Reply:
Wrong PR, I think this one should get in and then we open a specific one for podman cp.

Reply:
Fair point.